syft/formats/spdx22json/decoder_test.go

```diff
@@ -80,6 +80,15 @@ func TestSPDXJSONDecoder(t *testing.T) {
 						if p.Name == pkgName {
 							continue packages
 						}
+						switch p.Type {
+						case pkg.JavaPkg:
+							if p.Metadata.(pkg.JavaMetadata).VirtualPath != "acquired package info from installed java archive: /usr/lib/jvm/java-8-openjdk-amd64/jre/lib/ext/nashorn.jar" {
+								assert.NoError(t, fmt.Errorf("Mismatch in jar for package: %s. "+
+									"Expected `acquired package info from installed java archive: "+
+									"/usr/lib/jvm/java-8-openjdk-amd64/jre/lib/ext/nashorn.jar` and "+
+									"received VirtualPath: %s ", pkgName, p.Metadata.(pkg.JavaMetadata).VirtualPath))
+							}
```

Comment:
You could just do an `assert.Equal(t, "acquired package info from installed java archive: /usr/lib/jvm/java-8-openjdk-amd64/jre/lib/ext/nashorn.jar", p.Metadata.(pkg.JavaMetadata).VirtualPath)`

Reply:
Thanks!